### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BlobFile_AcquireLease_Screenreader : Jaws unnecessarily announcing No button twice.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1949]
" Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing aster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; Win10;

Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Right click on blob and select "Acquire Lease" control and hit enter.
7. Verify that all the elements of "Acquire Lease" control are accessible and MAS Compliant.

​Actual:​
Jaws unnecessarily announcing No button twice.

**Same is observed for below scenarios:
1. Disk - Delete Disk :** 
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Delete Disk button and click on it.
7. "Delete Disk" Popup gets opened.
8. Verify that all the elements of "Delete Disk" Screen are accessible and MAS Compliant.

Expected:
Jaws should announce No button only once.

User Impact:
Screen readers user will get unnecessarily get information two times.

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx 

MAS Reference:
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference:
[1949_A11y_AzureToolsStorageExplorer_BlobFile_AcquireLease_Screenreader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3661267/1949_A11y_AzureToolsStorageExplorer_BlobFile_AcquireLease_Screenreader.pptx)

**Comment by JasonYeMSFT** [comment]
This seems to be related to the extra tab stop in electron. The 2nd announcement of the No button happens when the navigation goes one step beyond the actual No button, which is announced as blank in NVDA.

**Comment by Alexandria-R** [comment]
Approved for exclusion as low impact.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
